query_test.go

```diff
@@ -376,6 +377,14 @@ func TestGormLike_Initialize_TriggersLikingCorrectly(t *testing.T) {
 			assert.NoError(t, err)
 
 			var actual []ObjectA
+
+			sqlQuery := db.ToSQL(func(tx *gorm.DB) *gorm.DB {
+				q := testData.query(tx).Where(testData.filter).Find(&actual)
+				return q
+			})
+
+			fmt.Printf("query: %s\n", sqlQuery)
```

Comment:
Sneaky debug statement snuck into the review ;)

Question: Wouldn't .Debug have been easier?